src/pages/RedirectToDownload.tsx

```diff
@@ -0,0 +1,14 @@
+import { useParams } from 'react-router-dom'
+import { DIRECT_DOWNLOAD_URL } from '../constants'
+
+const RedirectToDownload = (): null => {
+  const { hash } = useParams<{ hash: string }>()
+
+  // React router can redirect only withing the app itself, that is why we need to assing window.location
+  // https://knowbody.github.io/react-router-docs/api/Redirect.html
+  window.location.assign(`${DIRECT_DOWNLOAD_URL}${hash}`)
```

Comment:
Would it be better to use `replace` instead of `assign` in order to avoid saving the intermediate url in history?

Reply:
```suggestion
  window.location.replace(`${DIRECT_DOWNLOAD_URL}${hash}`)
```